Analysis/Tasks/PWGHF/HFLcK0sPCandidateSelector.cxx

```diff
@@ -0,0 +1,400 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HFLcK0spCandidateSelector.cxx
+/// \brief Lc --> K0s+p selection task.
+///
+/// \author Chiara Zampolli <Chiara.Zampolli@cern.ch>, CERN
+
+/// based on HFD0CandidateSelector.cxx
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "AnalysisDataModel/HFSecondaryVertex.h"
+#include "AnalysisDataModel/HFCandidateSelectionTables.h"
+#include "AnalysisTasksUtils/UtilsDebugLcK0Sp.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::aod::hf_cand_casc;
+
+static const int nPtBins = 8;
+static const int nCutVars = 8;
+//temporary until 2D array in configurable is solved - then move to json
+//mK0s(GeV)     mLambdas(GeV)    mGammas(GeV)    ptp     ptK0sdau     pTLc     d0p     d0K0
+constexpr double cuts[nPtBins][nCutVars] = {{0.008, 0.005, 0.1, 0.5, 0.3, 0.6, 0.05, 999999.},  // 1 < pt < 2
```

Comment:
This would need to be turned into a configurable, like in other HF tasks.

Reply:
Yes, but I would leave this to another PR. This is already big, and I might not have time to work on it in the next couple of weeks.

Reply:
Sure, that works.